devTools/cardCreator/cards_to_import/NonDuelyst/NomadSparringMaster.minion.card.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "Nomad Sparring Master",
+  "description": "Provoke, On Defend: Your general gains 3 Health",

```

Comment:
If you want this to work "on counter attack", we have a new tag for this.  see SunSeer / phantom as an example.